README.md

```diff
@@ -11,4 +11,12 @@ You can find the Step repositories which we maintain and support by searching fo
 
 ## Sample iOS Apps
 
-You can find our Sample App repositories by searching for [sample-](https://github.com/bitrise-io?utf8=%E2%9C%93&query=sample-) on our GitHub account page (github.com/bitrise-io).
```

Comment:
Keep this as "show all" `[sample-](https://github.com/bitrise-io?utf8=%E2%9C%93&query=sample-)`